src/legacy/core_plugins/console/public/quarantined/_app.scss

```diff
@@ -1,6 +1,16 @@
 // TODO: Move all of the styles here (should be modularised by, e.g., CSS-in-JS or CSS modules).
 #consoleRoot {
-  height: 100%;
+  display: -webkit-box;
```

Comment:
are the vendor prefixes necessary? i see in `package.json` that we do have [autoprefixer](https://github.com/postcss/autoprefixer) as a dependency. maybe someone from design or operations can confirm.